can/io/blf.py

```diff
@@ -119,8 +119,13 @@ def timestamp_to_systemtime(timestamp: float) -> TSystemTime:
 
 def systemtime_to_timestamp(systemtime: TSystemTime) -> float:
     try:
+        year_adjusted = (
+            systemtime[0]
+            if systemtime[0] >= 1971
+            else systemtime[0] + (1971 - systemtime[0])
+        )
         t = datetime.datetime(
-            systemtime[0],
+            year_adjusted,
```

Comment:
```suggestion
            max(systemtime[0], 1971),
```

This should be sufficient?

Reply:
Also, shouldn't it be 1970? 🤔

Reply:
Yes, that were my thoughts too. According to [the docs of `datetime.datetime`](https://docs.python.org/3/library/datetime.html#datetime.datetime), small numbers (e.g., year 1960) should be possible though.